docs/zikula_theme.yaml

```diff
@@ -0,0 +1,7 @@
+zikula_theme:
+    script_position: head
+    asset_manager:
+        combine: false
+        lifetime: '1 day'
+        compress: true
+        minify: true
```

Comment:
this is currently in the /docs dir to show the structure of the package file if needed. This would ultimately need to be part of the docs.